src/components/VContentSwitcher/VSearchTypeButton.vue

```diff
@@ -92,14 +92,7 @@ export default defineComponent({
       }
     })
     const buttonLabel = computed(() => {
-      const labelKey = {
-        image: 'search-type.image',
-        audio: 'search-type.audio',
-        all: 'search-type.all',
-        video: 'search-type.video',
-        model_3d: 'search-type.model_3d',
-      }[props.activeItem]
-      return i18n.t(labelKey)
```

Comment:
yay!

Reply:
Wasn't this because the context couldn't be identified correctly for the .POT file otherwise?

Reply:
I think this wasn't working anyways, because I don't see the `search-type` related changes in the `pot` file :(